README.md

```diff
@@ -31,6 +31,8 @@ This functionality is in beta and is subject to change. The design and code is l
 | `annotations`                 | annotations to be applied to the StatefulSet and Service                                                                                                                                | ``                                                                                                           |
 | `podAnnotations`                 | annotations to be applied to the StatefulSet pod Template                                                                                                                                | ``                                                                                                           |
 | `elasticsearch.enabled`                 | Enable ElasticSearch deployment as part of the Zeebe Cluster                                                                                                                                | `true`                                                                                                           |
+| `elasticsearch-curator.enabled`         | Enables the Elasticsearch curator to regularly clean up your Elastic indexes
+| `true`
```

Comment:
Lets not enabled it by default, because this will delete users data which they might not expect if they just update there helm charts